packages/api/blocks-api/src/blocks/block.ts

```diff
@@ -37,16 +37,19 @@ export interface ChildBlockInfo {
     name: string;
 }
 export interface BlockIndexData {
-    damFileIds?: string[];
+    dependencies?: Array<{
+        targetEntityName: string;
+        id?: string;
```

Comment:
This shouldn't be optional. Instead, the block implementing the dependency shouldn't add the array entry